components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -820,12 +820,10 @@ def _get_prepared_image(request, iid, server_id=None, conn=None,
         if reverses is not None and invert_flags is not None:
             invert_flags = [z[0] if z[0] is not None else z[1] for z in
                             zip(invert_flags, reverses)]
-
-    # more rendering settings per channel: quantization maps
-    # just applied, not saved at the moment
-    if 'quant_maps' in r:
         try:
-            img.setQuantizationMaps(json.loads(r['quant_maps']))
+            # quantization maps (just applied, not saved at the moment)
+            qm = [m['quantization'] for m in json.loads(r['maps'])]
```

Comment:
Final tweak (sorry)...
Use ``` m.get('quantization') ``` instead of ```m['quantization']``` since we can't assume that it's there.
If it's missing for a single channel, we don't want it to fail for All other channels. E.g. support this:
```
&maps=[{"inverted":{"enabled":false}},{"quantization":{"family":"linear","coefficient":0.1},"inverted":{"enabled":false}}]
```